2a_model/src/models/0_baseline_LSTM/config.yml

```diff
@@ -1,45 +1,8 @@
-out_dir: "../../../out/models"
 exp_name: "0_baseline_LSTM"
 
-#### for flow modeling, seg_width should be removed from x_vars
```

Comment:
I'm leaving only the experiment-specific options in this file (e.g., "exp_name", "y_vars") because its the "experiment-specific" config file that extends the "base" config file.